### PR TITLE
adds logging to mayhem bottle and blood frenzy

### PR DESCRIPTION
**Pull request by imsxz**
:cl: imsxz
admin: Using mayhem bottles and going under their effects is now logged.
/:cl:

let me know if i did this incorrectly, i just felt that logging should exist for this

### CODE REVIEW

File: code/game/objects/effects/mines.dm [hunk 1]
```diff
@@ -129,6 +129,7 @@
 		new /datum/hallucination/delusion(victim, TRUE, "demon",duration,0)
 
 	var/obj/item/twohanded/required/chainsaw/doomslayer/chainsaw = new(victim.loc)
+	add_logs(victim, "has entered a blood frenzy")
```
Comment (SpaceManiac) [inline code/game/objects/effects/mines.dm]: I don't think `add_logs` is the right proc to use here - looks like it's intended for attack logs with both an attacker and a target involved. Maybe just `log_attack` directly if you're sure you want it to go to the attack log?

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/mines.dm (hunk 1 above)
Comment (imsxz) [reply code/game/objects/effects/mines.dm]: ah thanks. just wanted to have it show somewhere so admins can easily see when the person gained and lost their murderbone privilege from the frenzy.

**Comment by Kromgar** [comment]
>that feel when i get banned for using chaos in a bottle as non antag now

**Review by SpaceManiac — CHANGES_REQUESTED** [review]
After more close review, `add_logs` is fine, you want something like:

```dm
add_logs(victim, null, "entered a blood frenzy")
add_logs(victim, null, "exited a blood frenzy")
add_logs(user, null, "activated a bottle of mayhem", src)
```

Note the definition of `add_logs`:
```dm
/proc/add_logs(mob/user, mob/target, what_done, object=null, addition=null)
```

**Comment by imsxz** [comment]
ree

**Review by SpaceManiac — APPROVED** [review]
Alright, looks good.

**Comment by ExcessiveUseOfCobblestone** [comment]
Not sure a CL is needed but if you're going to add one remind people that it's currently allowed regardless of antag or not.

**Comment by SpaceManiac** [comment]
No CL needed I think, basically only relevant for log-diving. Admins thinking any admin log message is punishable is their problem.

**Pull request merged**